socorro/app/socorro_app.py

```diff
@@ -429,21 +370,14 @@ class App(SocorroApp):
         default=8125,
         reference_value_from='resource.statsd'
     )
+    required_config.metricscfg.add_option(
```

Comment:
Because there's an aggregate and that ends up as `metrics`.

With logging, we use `logging` for configuration and then `logger` as the aggregate. When I did the initial metrics stuff a while back, I really struggled with equivalent names and then settled on `metricscfg` for the namespace because I couldn't think of a better one.